client/src/modules/reports/generate/cashflow/cashflow.html

```diff
@@ -0,0 +1,60 @@
+<bh-report-preview
+  ng-if="ReportConfigCtrl.previewGenerated"
+  source-document="ReportConfigCtrl.previewResult"
+  on-clear-callback="ReportConfigCtrl.clearPreview()"
+  on-save-callback="ReportConfigCtrl.requestSaveAs()">
+</bh-report-preview>
+
+<div ng-show="!ReportConfigCtrl.previewGenerated">
+  <div class="row">
+    <div class="col-md-12">
+      <h3 class="text-capitalize" translate>REPORT.CASHFLOW.TITLE</h3>
+      <p class="text-info" translate>REPORT.CASHFLOW.DESCRIPTION</p>
+    </div>
+  </div>
+
+  <div class="row" style="margin-top : 10px">
+    <div class="col-md-6">
+      <div class="panel panel-default">
+        <div class="panel-heading">
+          <span translate>REPORT.UTIL.OPTIONS</span>
+        </div>
+
+        <div class="panel-body">
+          <form name="ConfigForm" bh-submit="ReportConfigCtrl.preview(ConfigForm)" novalidate>
+            <!-- date interval  -->
+            <bh-date-interval
+              date-from="ReportConfigCtrl.reportDetails.dateFrom"
+              date-to="ReportConfigCtrl.reportDetails.dateTo"
+              required="true">
+            </bh-date-interval>
+
+            <!-- by week  -->
+            <div class="checkbox">
```

Comment:
It might be good to include some explanation text about what a weekly checkbox does.  You can use a `<p class="help-block"></p>` tag to add this text under the `<div class="checkbox></div>`.

For reference, [here is how the Open Debtors report](https://github.com/IMA-WorldHealth/bhima-2.X/blob/master/client/src/modules/reports/generate/open_debtors/open_debtors.html#L89) accomplishes this.